src/providers/twitch/TwitchChannel.cpp

```diff
@@ -365,6 +365,7 @@ void TwitchChannel::sendMessage(const QString &message)
     // Do last message processing
     QString parsedMessage = app->emotes->emojis.replaceShortCodes(message);
 
+    parsedMessage.replace("\u200D", "\U000E0002");
```

Comment:
I believe it would be better to use named constants instead of magic values here

Reply:
Sure!

Reply:
Also, for context, it would make sense to drop a comment with links to https://github.com/Chatterino/chatterino2/issues/3384 and https://github.com/twitchdev/issues/issues/307 somewhere in the code